Packs/Darktrace/IncidentTypes/DarktraceIncidentType.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
the same comment as incidentfield. 
remove the list before the incident type content.
Also, its better to rename the file as the incidentfield id.

Reply:
Adjusted! Let me know if this looks correct -- new filename is incidenttype-Darktrace_Model_Breach.json

Reply:
Looks good, the same comment regarding incident-fields and their fromversion key applies here too.

Reply:
fromversion should now be resolved